### PR TITLE
Mm 11 creating blank funfact component and route

### PR DESCRIPTION
**Pull request by rosalievan**
Ticket 11 - creating funfact component and adding it to router

### CODE REVIEW

File: src/components/funFactPage/funFactPage.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+export const FunFactPage: React.FunctionComponent = () => {
+    return (
+      <main>
+        <h1>FunFactPage</h1>
+        <p>FunFactPage</p>
+      </main>
+    );
+  };
```
Comment (TimLeach635) [inline src/components/funFactPage/funFactPage.tsx]: See how GitHub has added a little ⛔ at the bottom of the file here? This is to indicate that the file doesn't end with a single empty line, which is usually preferred. Could you add one here please?

File: src/components/funFactPage/funFactPage.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
```
Comment (TimLeach635) [inline src/components/funFactPage/funFactPage.tsx]: 1. The convention is this project is to name our component files with a capital letter in the beginning, like `FunFactPage.tsx` and `FunFactPage.scss`. Sadly, Windows file systems are case insensitive, so _this can't be fixed_ without having to delete a whole bunch of stuff and recreate it 😬 you can leave this one, but please try to remember in future! (The folder name, `funFactPage`, is cased correctly 😊 although you may be able to fix the casing by renaming this file with a capital (`funFactPage.tsx -> FunFactPage.tsx`), then renaming the folder to `funFacts`, and committing them both together, so try that! If it doesn't work don't worry)
2. Don't create an empty `.scss` file just as a placeholder, we may not use it! Please delete that file :blush:

**Review by TimLeach635 — CHANGES_REQUESTED** [review]
Some style changes!

File: src/components/funFactPage/funFactPage.tsx (hunk 1 above)
Comment (rosalievan) [reply src/components/funFactPage/funFactPage.tsx]: Done

File: src/components/funFactPage/funFactPage.tsx (hunk 2 above)
Comment (rosalievan) [reply src/components/funFactPage/funFactPage.tsx]: Done - I think the foldername update funFacts worked so I hope that comes through

**Review by TimLeach635 — APPROVED** [review]

**Pull request merged**